spec/features/workflow/publish_spec.rb

```diff
@@ -41,4 +75,59 @@ def and_the_content_is_shown_as_published
   def and_there_is_a_history_entry
     expect(page).to have_content(I18n.t!("documents.history.entry_types.published"))
   end
+
+  def and_i_receive_a_confirmation_email
+    tos = ActionMailer::Base.deliveries.map(&:to)
+    message = ActionMailer::Base.deliveries.first
+
+    publish_time = @publish_date.strftime("%l:%M%P")
+    publish_date = @publish_date.strftime("%d %B %Y")
+    publish_user = current_user.name
+
+    expect(tos).to match_array [[@creator.email], [current_user.email]]
+    expect(message.body).to include("https://www.test.gov.uk/news/banana-pricing-updates")
+    expect(message.body).to include(document_path(@edition.document))
+
+    expect(message.subject).to eq(I18n.t("publish_mailer.publish_email.subject.published",
+                                         title: @edition.title))
+
+    expect(message.body).to include(I18n.t("publish_mailer.publish_email.details.publish",
+                                           time: publish_time,
+                                           date: publish_date,
+                                           user: publish_user))
+  end
+
+  def then_i_receive_an_email_about_the_major_change
+    message = ActionMailer::Base.deliveries.first
+
+    publish_time = @publish_date.strftime("%l:%M%P")
+    publish_date = @publish_date.strftime("%d %B %Y")
+    publish_user = current_user.name
+    expect(message.body).to include(@edition.change_note)
+
+    expect(message.subject).to eq(I18n.t("publish_mailer.publish_email.subject.update",
+                                         title: @edition.title))
+
+    expect(message.body).to include(I18n.t("publish_mailer.publish_email.details.update",
+                                           time: publish_time,
+                                           date: publish_date,
+                                           user: publish_user))
+  end
+
+  def then_i_receive_an_email_about_the_minor_change
```

Comment:
As this is very similar to `#then_i_receive_an_email_about_the_major_change` I wonder if it's enough here just to test that the email includes the minor update text?

Reply:
Sure - changed :-).